tracer/src/Datadog.Tracer.Native/dd_profiler_constants.h

```diff
@@ -65,13 +65,15 @@ const shared::WSTRING include_assemblies[]{
 
 // Note that this list should be kept in sync with the values in shared/src/Datadog.Trace.ClrProfiler.Native/util.h
 const shared::WSTRING default_exclude_assemblies[]{
+    WStr("csc.exe"),
```

Comment:
Is there a way to allow for this to be configurable as well?

Reply:
This is just the _default_ list of assemblies, so customers can override this list, like we do here: https://github.com/DataDog/dd-trace-dotnet/blob/cc814d258f77ad8484c176465cac421ed3e87e99/tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs#L290

Reply:
Ahhhhh thanks :)